jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -753,6 +754,16 @@ function configure_jbpm_cache() {
         </cache-container>\n<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#g' ${CONFIG_FILE}
 }
 
+function configure_jbpm_cache_modules() {
```

Comment:
is there a reason to keep `configure_jbpm_cache` function and this one separated?
if none of them is used anywhere else, can't we merge them into a single one?

Reply:
One change the cache configuration, the second the classpath configuration and are separated for "unit" test

Reply:
ok

Reply:
Ok, joined